src/components/Stack/Stack.tsx

```diff
@@ -32,33 +32,31 @@ export interface StackProps {
   distribution?: Distribution;
 }
 
-export class Stack extends React.PureComponent<StackProps, never> {
-  static Item = Item;
-
-  render() {
-    const {
-      children,
-      vertical,
-      spacing,
-      distribution,
-      alignment,
-      wrap,
-    } = this.props;
-
-    const className = classNames(
-      styles.Stack,
-      vertical && styles.vertical,
-      spacing && styles[variationName('spacing', spacing)],
-      distribution && styles[variationName('distribution', distribution)],
-      alignment && styles[variationName('alignment', alignment)],
-      wrap === false && styles.noWrap,
-    );
-
-    const itemMarkup = elementChildren(children).map((child, index) => {
-      const props = {key: index};
-      return wrapWithComponent(child, Item, props);
-    });
-
-    return <div className={className}>{itemMarkup}</div>;
-  }
-}
+export const Stack = memo(function Stack({
```

Comment:
Per my thoughts in https://github.com/Shopify/polaris-react/issues/2062, I'm not a fan of us using memo everywhere, at it feels like our usage of that and PureComponent has been very cargo-culty and we've been doing it without measuring its impact.

What do you think to removing the `memo()` wrapper?

Reply:
Left my thoughts in this [post](https://github.com/Shopify/polaris-react/pull/2539#issuecomment-571275058)

Reply:
Thanks for reckons :) Happy to say removing PureComponent / React.memo bits can be kept out of the hookification and they can be looked at as a separate phase.